test/functional/test_framework/test_node.py

```diff
@@ -931,8 +931,9 @@ def __init__(self, rpc):
     def __getattr__(self, name):
         return getattr(self.rpc, name)
 
-    def createwallet_passthrough(self, *args, **kwargs):
-        return self.__getattr__("createwallet")(*args, **kwargs)
+    def rpc_passthrough(self, name, *args, **kwargs):
+        """Directly pass through the RPC, avoiding any wrapped method."""
+        return self.__getattr__(name)(*args, **kwargs)
```

Comment:
> avoiding any wrapped method.

Avoiding any of the below 4 wrapped methods?

> add a more general (currently
unused) method rpc_passthrough, which could be used in the future, if
there really is need for it.

It seems to be effectively same as calling the instance returned by `__getattr__ ` above, while improving readability. I don't think it hurts to add this but I sense it will not end up being used if there are not already usages of it in the framework.